appinfo/info.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<info>
+	<id>files_mediaviewer</id>
+	<name>Mediaviewer</name>
+	<summary>Explore, show your image collections</summary>
+	<description>Mediaviewer adds image and view viewing capabilities to files view and public links.</description>
```

Comment:
Q:
- `Mediaviewer adds image and view viewing capabilities`
What are image and view viewing capabilities.
I guess you meant `Mediaviewer adds image viewing capabilities`

- From the repro title `files mediaviewer` I expect that there are also videos included which does not seem to be.
In case this is image viewing only, the repro and the titles should be renamed properly.
In case video viewing is part of the capabilities - even implemented later - this should be documented properly.